configs/jsactions/rollup.config.js

```diff
@@ -81,20 +81,12 @@ export default async args => {
                                   }
                               }
 
-                              const destinationFolder = join(cwd, "tests/testProject/", jsActionTargetFolder);
-                              const destinations = [
-                                  destinationFolder,
-                                  ...[
-                                      process.env.MX_PROJECT_PATH
-                                          ? join(process.env.MX_PROJECT_PATH, jsActionTargetFolder)
-                                          : undefined
-                                  ]
-                              ];
-
-                              destinations.filter(Boolean).forEach(destination => {
+                              // this is helpful to copy the files and folders to a test project path for dev/testing purposes.
```

Comment:
I think it's nice to keep. I know some people are of the opinion that all code should be self documenting. I think some comments such as this can be of value to developers very new to the codebase.

Reply:
If you think this is worthwhile, let's just keep it